android/src/main/java/com/fidelreactlibrary/adapters/abstraction/VerificationSetupAdapter.java

```diff
@@ -0,0 +1,9 @@
+package com.fidelreactlibrary.adapters.abstraction;
+
+import com.fidelapi.entities.CardVerificationConfiguration;
+import com.facebook.react.bridge.ReadableMap;
+
+public interface VerificationSetupAdapter {
```

Comment:
💅 Again, to be consistent with the terms that we use, I suggest renaming this to `VerificationConfigurationAdapter`, as we already have a `CardVerificationConfiguration` class used for "configuring" card verification, not "setup" :).